emukit/core/loop/stopping_conditions.py

```diff
@@ -31,4 +37,31 @@ def should_stop(self, loop_state: LoopState) -> bool:
         :param loop_state: Object that contains current state of the loop
         :return: True if maximum number of iterations has been reached
         """
-        return loop_state.iteration >= self.i_max
+        status = loop_state.iteration >= self.i_max
+        if status is True:
```

Comment:
You can use a more informative name, like `should_stop`, and just do `if should_stop`, which reads nicer than `is status is True`